x-pack/test/reporting/api/bwc_generation_urls.js

```diff
@@ -33,6 +33,26 @@ export default function ({ getService }) {
       });
     });
 
+    describe('6_4', () => {
+      before(async () => {
+        await reportingAPI.deleteAllReportingIndexes();
+      });
+
+      /*The URL being tested was captured from release 6.4 and then the layout section was removed to test structure before
+        preserve_layout was introduced. See https://github.com/elastic/kibana/issues/23414 */
+      it('multiple jobs posted', async () => {
+        const reportPaths = [];
+        reportPaths.push(await reportingAPI.postJob(GenerationUrls.PDF_PRINT_DASHBOARD_PRE_6_2));
+
+        await reportingAPI.expectAllJobsToFinishSuccessfully(reportPaths);
```

Comment:
Since this is only posting one job, can simplify this a bit:

```
it('job posted successfully', async () => {
  const path = await reportingAPI.postJob(GenerationUrls.PDF_PRINT_DASHBOARD_PRE_6_2)
  await this.waitForJobToFinish(path);
  const stats = await usageAPI.getUsageStats();
  reportingAPI.expectCompletedReportCount(stats, 1);
}).timeout(500000);
```

I dropped the timeout since only one job is being posted, and combined the two tests into a single `it`.  Seemed like overkill to split it out for one job.